integration/README.md

```diff
@@ -4,36 +4,43 @@
 
 The integration/e2e tests have the following prerequisites:
 
-  * an environment variable `CLUSTER_MANIFEST_FILE` pointing to the cluster manifest. Cluster manifests can be produced with the help of the templates found under `templates`. For example:
+  * an environment variable `CLUSTER_MANIFEST_FILE` pointing to the cluster manifest. Cluster manifests can be produced with the help of the templates found under [`templates`](../templates). For example:
+  ```bash 
+export AWS_REGION=us-east-1
+export AWS_SSH_KEY_NAME=capi
+export CONTROL_PLANE_MACHINE_COUNT=3
+export WORKER_MACHINE_COUNT=3
+export AWS_CREATE_BASTION=false
+export AWS_PUBLIC_IP=false
+export AWS_CONTROL_PLANE_MACHINE_FLAVOR=t3.large
+export AWS_NODE_MACHINE_FLAVOR=t3.large
+export CLUSTER_NAME=test-ci-cluster
+clusterctl generate cluster ${CLUSTER_NAME} --from "templates/cluster-template-aws.yaml" --kubernetes-version 1.27.0 > cluster.yaml
+export CLUSTER_MANIFEST_FILE=$PWD/cluster.yaml
   ```
-    export AWS_REGION=us-east-1
-    export AWS_SSH_KEY_NAME=capi
-    export CONTROL_PLANE_MACHINE_COUNT=3
-    export WORKER_MACHINE_COUNT=3
-    export AWS_CREATE_BASTION=false
-    export AWS_PUBLIC_IP=false
-    export AWS_CONTROL_PLANE_MACHINE_FLAVOR=t3.large
-    export AWS_NODE_MACHINE_FLAVOR=t3.large
-    export CLUSTER_NAME=test-ci-cluster
-    clusterctl generate cluster ${CLUSTER_NAME} --from "templates/cluster-template-aws.yaml" --kubernetes-version 1.25.0 > cluster.yaml
-    export CLUSTER_MANIFEST_FILE=$PWD/cluster.yaml
-  ```
-
+> NOTE: AWS_SSH_KEY_NAME is name of ssh key in AWS that you are plan to use, if you don't have one yet refer
```

Comment:
nit: the name

Reply:
@maci3jka please revert these formatting changes, they are not needed for this PR